eaf_pdf_page.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, page, page_index, is_pdf, clip=None):
 
         self.has_annot = page.first_annot
         self.hovered_annot = None
+        self._current_match_color = QColor("#f28100").getRgbF()[0:3]
```

Comment:
Don't hardcode here, please follow Emacs theme

Reply:
below is the origianl defintion or the color  :

```
             if quads == current_quads:
                    qcolor = QColor("#f28100")
                    annot.set_colors(stroke=qcolor.getRgbF()[0:3])
```
I move the `qcolor = QColor("#f28100") `  to __init__ to avoid creating a constant QColor every time the frame updates, I think this modification will not affect the logic of the orignal  codes, my be I am wrong, should I revert this modification back or add new features to follow Emacs theme?

Reply:
I have revert the codes related to color setting, thanks for review